Add space ruin: abandoned engi sat

warriorstar-orion opened the pull request:
## What Does This PR Do

"This NT satellite, previously used by engineering departments of ships operating in the local sector, was abandoned after forecasts of extreme meteor activity. The satellite was evacuated and decommissioned, with all of its equipment relocated. Despite the actual damage from the meteors being minimal, the satellite has never been reclaimed by Nanotrasen."

The only things of value on the ruin are the hidden vodka and contraband posters, and the cell charger. No belts, no insuls, nothing. The suit storage units are empty, as are all of the equipment lockers. 

<details><summary>Notes for map reviewers</summary>

This DMM contains a lot of strange-looking varedits that I'm using to accentuate the feeling of a completely abandoned. Notably:

- Some light bulbs are broken in their brackets, requiring an `icon_state` and `status` change,
- All closets are base types that have subtype icons/data so that they are empty when spawned, rather than spawning with their typical equipment.
- A torn "SPACE" sign is just a renamed torn poster.

Additionally, the map uses modern airlock helpers. I set them to engineering for realism.

</details>

## Why It's Good For The Game
More space ruins are good, I thought this might be a cool nod to Para history, and bringing in an old NT property as a ruin makes it easy to incorporate into lore.

## Images of changes
In DMM:
![2023_09_26__13_23_15__paradise dme  abandoned_engi_sat dmm  - StrongDMM](https://github.com/ParadiseSS13/Paradise/assets/59303604/66815d4c-033d-4708-a00a-381539cc6517)

In-game, full dark,

![2023_09_26__13_26_58__](https://github.com/ParadiseSS13/Paradise/assets/59303604/9d816f5f-d1b1-4471-9cc4-13f04b7e07ad)

In-game, APC powered

![2023_09_26__13_26_24__](https://github.com/ParadiseSS13/Paradise/assets/59303604/317d7361-318f-441c-b431-13482c9925b0)


## Testing
Spawned ruin in, teleported, tested APC behavior, tested item interactions.

## Changelog
:cl:
add: Adds a space ruin, the abandoned NT Engineering satellite
/:cl:

AffectedArc07 commented:
A good homage to an awful time

Bormac commented:
Needs cult references. Thats like 90% of what Engi sat was used for then 9% running from the nuke. 1% actual engi stuff. 
But really this is good.

AffectedArc07 commented on code/datums/ruins/space_ruins.dm:
```diff
@@ -326,3 +326,11 @@
+
+/datum/map_template/ruin/space/abandoned_engi_sat
+	id = "abandoned_engi_sat"
+	suffix = "abandoned_engi_sat.dmm"
```
Yeah I've no idea why this is failing CI.

I checked this in the code and paths and its cased right.

warriorstar-orion replied:
Whoops, fixed

ATP-Engineer commented:
Add catwalk tiles on the interior maybe to look cooler?

Henri215 reviewed (approved):
LGTM

S34NW reviewed (commented):
![image](https://github.com/ParadiseSS13/Paradise/assets/12197162/42d1a6fc-44a3-4db8-b40c-b4a4a99d65e1)
All your light buttons are one dir, even though they have dirs.

![image](https://github.com/ParadiseSS13/Paradise/assets/12197162/f5d212df-aeda-420b-9ecd-4f48591ff984)
This is going to make me cry if not centered

![image](https://github.com/ParadiseSS13/Paradise/assets/12197162/151240ad-9cc6-4f75-bea1-f018696c0736)
Pipes under walls :( Although I don't see a better way so it will do here

![image](https://github.com/ParadiseSS13/Paradise/assets/12197162/c6c4bfd7-ca2b-4194-b87e-77f35c1d1f44)
Console faces a wall

![image](https://github.com/ParadiseSS13/Paradise/assets/12197162/8caf109e-61a5-4336-bc1a-63d487a1b268)
console south facing

![image](https://github.com/ParadiseSS13/Paradise/assets/12197162/87351e30-11bc-4724-a63f-4bb2a4850e6f)
This one isn't an issue, but APCs autoname themselves based on their area so this var edit is kinda pointless

warriorstar-orion commented:
> ![image](https://user-images.githubusercontent.com/12197162/271841082-42d1a6fc-44a3-4db8-b40c-b4a4a99d65e1.png) All your light buttons are one dir, even though they have dirs.
> 

fixed

> ![image](https://user-images.githubusercontent.com/12197162/271841115-f5d212df-aeda-420b-9ecd-4f48591ff984.png) This is going to make me cry if not centered
>
fixed

> ![image](https://user-images.githubusercontent.com/12197162/271841159-151240ad-9cc6-4f75-bea1-f018696c0736.png) Pipes under walls :( Although I don't see a better way so it will do here

i modernized most of the ruin but i figured in this case it's fine to have the pipes as they were on the original sat, before the mapping standards were really formulated

> ![image](https://user-images.githubusercontent.com/12197162/271841190-c6c4bfd7-ca2b-4194-b87e-77f35c1d1f44.png) Console faces a wall
> 
> ![image](https://user-images.githubusercontent.com/12197162/271841196-8caf109e-61a5-4336-bc1a-63d487a1b268.png) console south facing
> 

both consoles fixed

> ![image](https://user-images.githubusercontent.com/12197162/271841354-87351e30-11bc-4724-a63f-4bb2a4850e6f.png) This one isn't an issue, but APCs autoname themselves based on their area so this var edit is kinda pointless

fixed!

Christasmurf commented:
![beautifulasilostyou](https://github.com/ParadiseSS13/Paradise/assets/25437893/603cf681-31ce-47d9-bb45-f5024db21f5a)
